### PR TITLE
Fix react-native 0.40 imports

### PR DESCRIPTION
**Pull request by farwayer**
https://github.com/facebook/react-native/releases/tag/v0.40.0

### CODE REVIEW

File: example/ios/example/AppDelegate.m [hunk 1]
```diff
@@ -8,54 +8,29 @@
...
                                                    launchOptions:launchOptions];
+  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
 
   self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
-  
-  [Fabric with:@[[Crashlytics class]]];
```
Comment (corymsmith) [inline example/ios/example/AppDelegate.m]: Is there any reason why you're removing this? Otherwise this example project won't work

**Comment by corymsmith** [comment]
FYI, I just merged #83 which includes updates to the imports and released v0.4.0 to NPM.

**Comment by farwayer** [comment]
Oh, sorry missed this PR

**Comment by corymsmith** [comment]
No worries, I took a while to merge it in

**Pull request closed without merging**